packages/ses/test/make-safe-evaluator.test.js

```diff
@@ -4,17 +4,29 @@ import test from 'ava';
 import { makeSafeEvaluator } from '../src/make-safe-evaluator.js';
 
 test('safeEvaluate - default (non-sloppy, no moduleLexicals)', t => {
-  t.plan(6);
+  //t.plan(6);
```

Comment:
```suggestion
  // t.plan(6);
```

Or restore assertion with corrected value?